vscode-wpilib/src/builtintools.ts

```diff
@@ -18,33 +18,31 @@ interface IEnumerateResult {
 class VbsToolRunner implements IToolRunner {
   private toolScript: string;
   private name: string;
+  private preferences: IPreferencesAPI;
 
-  public constructor(toolScript: string, name: string) {
+  public constructor(toolScript: string, name: string, preferences: IPreferencesAPI) {
     this.toolScript = toolScript;
     this.name = name;
+    this.preferences = preferences;
   }
 
-  public runTool(): Promise<boolean> {
+  public async runTool(): Promise<boolean> {
+    const wp = await this.preferences.getFirstOrSelectedWorkspace();
     return new Promise<boolean>((resolve, _reject) => {
-      if (getIsWindows()) {
-        // If windows, run the vbs script
-        cp.exec(`wscript.exe ${this.toolScript}`, (err) => {
-          if (err) {
-            resolve(false);
-          } else {
-            resolve(true);
-          }
-        });
-      } else {
-        // Unix, run as python
-        cp.exec(`python ${this.toolScript}`, (err) => {
-          if (err) {
-            resolve(false);
-          } else {
-            resolve(true);
-          }
-        });
+      let cmd = getIsWindows() ? `wscript.exe ${this.toolScript} ` : `python ${this.toolScript}`;

```

Comment:
Doesn't really matter, but the wscript one has an extra space in the generated command line.